.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Generated from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cudf/labels
+
+cuDF (Python):
+  - python/
+  - notebooks/
+  
+libcudf:
+  - cpp/
+
+CMake:
+  - **/CMakeLists.txt
+  - **/cmake/
+  
+cuDF (Java):
+  - java/
+  
+Ops:
+  - .github/
+  - /ci/
+  - conda/
+  - **/Dockerfile
+  - **/.dockerignore
+  - docker/
```

Comment:
have these patterns been tested? I'm not sure that the file patterns in the `CODEOWNERS` file can be directly copied into this file. For instance, the [Common Examples](https://github.com/actions/labeler#common-examples) section says you would need to use `python/**/*` (not `python/`) to match any file (recursively) in the `python` directory. I think these will all need to be updated accordingly.